internal/testing/runchecks.sh

```diff
@@ -174,15 +181,18 @@ echo "Running wire diff..."
 
 echo
 echo "Running Go tests for sub-modules..."
+PATH="$PATH":"${PWD}"/internal/testing
```

Comment:
I wouldn't do this just to avoid prepending the dir to check_mod_tidy. Prefer explicit - also we do this elsewhere - starting local deps, listdeps, etc.

check_mod_tidy can take a dir and cd into it?

Reply:
Done.